resources/xcode_settings.bash

```diff
@@ -8,4 +8,5 @@ defaults write com.apple.dt.Xcode DVTTextEditorTrimWhitespaceOnlyLines -bool YES
 defaults write com.apple.dt.Xcode DVTTextIndentTabWidth -int 2
 defaults write com.apple.dt.Xcode DVTTextIndentWidth -int 2
 
-defaults write com.apple.dt.Xcode DVTTextPageGuideLocation -int 100
+defaults write com.apple.dt.Xcode DVTTextShowPageGuide -bool true
```

Comment:
Since we would be enforcing this maximum with a linter, I think it makes sense to show the page guide by default.